### PR TITLE
fix namespace issues

### PR DESCRIPTION
**Pull request by nhart**

### CODE REVIEW

File: transforms/handle_the_DC.xsl [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <xsl:template match="dc:dc">
         <xsl:copy>
             <xsl:apply-templates/>
-            <identifier type="hdl">
+            <dc:identifier xmlns:dc="http://purl.org/dc/elements/1.1/" type="hdl">
                 <xsl:value-of select="$handle_value"/>
             </identifier>
```
Comment (adam-vessey) [inline transforms/handle_the_DC.xsl]: Closing should probably also get the prefix. Actually... Adding the prefix here shouldn't be necessary at all, as it's the [default namespace for this stylesheet](https://github.com/discoverygarden/islandora_handle/pull/13/files#diff-61cb4fe92331890bd19753ca905d2b2fR4). Shrug?

File: transforms/handle_the_DC.xsl [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <xsl:template match="dc:dc">
         <xsl:copy>
             <xsl:apply-templates/>
-            <identifier type="hdl">
+            <dc:identifier xmlns:dc="http://purl.org/dc/elements/1.1/" type="hdl">
```
Comment (adam-vessey) [inline transforms/handle_the_DC.xsl]: Prefix is [already defined in the stylesheet](https://github.com/discoverygarden/islandora_handle/pull/13/files#diff-61cb4fe92331890bd19753ca905d2b2fR5), shouldn've have to declare again.

**Comment by jordandukart** [comment]
Are we concerned about a validation/update batch for existing people who may have busted DC?

**Comment by nhart** [comment]
@jordandukart I was thinking on this. Are we actually generating bad DC? This bit of code was previously throwing a 500 error. The bad DC was never making it back to Fedora. Thoughts?

**Comment by jordandukart** [comment]
Spose that's a valid point, given that this source DC probably isn't coming directly from Islandora right?

**Comment by nhart** [comment]
original dc - https://gist.github.com/nhart/bc42487e0ca9bff21cbf553a928b918b
dc missing default ns uri - https://gist.github.com/nhart/17855d4b39e2ddf99c0d288598db0c83

**Pull request merged**